task3.1.py

```diff
@@ -0,0 +1,11 @@
+'''Задайте список из нескольких чисел. Напишите программу, которая 
+найдёт сумму элементов списка, стоящих на нечётной позиции.
+Пример:
+- [2, 3, 5, 9, 3] -> на нечётных позициях элементы 3 и 9, ответ: 12'''
+
+count = 0
+lst = [1, 2, 3, 4, 5, 6, 7, 8, 90]
+for i in range(1, len(lst)):
+    if i % 2 == 0:
+        count += lst[i]
```

Comment:
```suggestion
for i in range(1, len(lst), 2):
    count += lst[i]
```
Если вы уж все равно указали старт функции `range`, так может быть указать заодно и шаг тогда не нужно будет писать условие